aepsych/models/variational_gp.py

```diff
@@ -113,4 +116,76 @@ def get_config_options(cls, config: Config, name: Optional[str] = None):
         options = super().get_config_options(config)
         if options["likelihood"] is None:
             options["likelihood"] = BernoulliLikelihood()
+        if options["inducing_size"] is None:
+            options["inducing_size"] = 10
+        return options
+    
+    
+class AxOrdinalGPModel(VariationalGP):
+    """
+    Convenience class for using a VariationalGP with an OrdinalLikelihood.
+    """
+
+    outcome_type = "ordinal"
+
+    def predict(self, xgrid):
+        with torch.no_grad():
+            post = self.posterior(xgrid)
+            fmean = promote_0d(post.mean.squeeze())
+            fvar = promote_0d(post.variance.squeeze())
+        return fmean, fvar
+
+    def predict_probs(self, xgrid):
```

Comment:
Let's combine `predict` and `predict_probs` into a single `predict` method with a `probability_space` argument like in `BinaryClassificationGP`.